.ci/update-github-io.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+set -e
+
+source ./.ci/util.sh
+
+if [[ -z $1 ]]; then
+  echo "version is not set"
+  echo "Usage: update-github-io.sh <version>"
+  exit 1
+fi
+
+TARGET_VERSION=$1
+echo TARGET_VERSION="$TARGET_VERSION"
+
+
+TAG=$(git describe --tags "$(git rev-list --tags --max-count=1)")
+echo TAG="$TAG"
+git checkout "$TAG"
+
+echo "Generating web site"
+mvn -e --no-transfer-progress site -Pno-validations -Dmaven.javadoc.skip=false
+
+echo "Clone by ssh only to avoid passwords on push ..."
+checkout_from git@github.com:checkstyle/checkstyle.github.io.git
```

Comment:
Do `cd . ci-temp `

Reply:
already there, https://github.com/checkstyle/checkstyle/pull/12118/files#diff-1e0d5171648bf085d51bf527b97f16adf65b43da541a135cf215d29435120161R26

Reply:
No.
You need to clone in that folder, not a root of checkstyle repo.

Cd is not persistent between GitHub action steps.

Reply:
`checkout_from` function creates the .ci-temp folder and clone in that only

Reply:
Ok, I didn't notice that this function is used, my bad